src/Workleap.Extensions.OpenAPI/TypedResult/ExtractSchemaTypeResultFilter.cs

```diff
@@ -121,6 +135,28 @@ internal static IEnumerable<ResponseMetadata> GetResponsesMetadata(Type returnTy
         return null;
     }
 
+    private static HashSet<int> ExtractResponseCodesFromAttributes(OperationFilterContext context)
+    {
+        HashSet<int> responseCodes = [];
+        foreach (var attribute in context.MethodInfo.CustomAttributes)
+        {
+            if (attribute.AttributeType.FullName is not (ProducesResponseTypeAttribute or SwaggerResponseTypeAttribute))
+            {
+                continue;
+            }
+
+            foreach (var argument in attribute.ConstructorArguments)
```

Comment:
Weird to do a foreach since an attribute can only have 1 httoCode. Is there a way to get the parameter by name?

Reply:
I was looking for a Named argument but they are not always present for example:
![image](https://github.com/gsoft-inc/wl-extensions-openapi/assets/158102624/b965a652-e8c1-4dc8-898f-805cf476d8f0)